enhancements/ingress/haproxy-max-connections-tuning.md

```diff
@@ -0,0 +1,350 @@
+---
```

Comment:
initial review looks good to me. I'll go over it again on Monday in more detail.